docs/field-details.asciidoc

```diff
@@ -453,12 +453,6 @@ example: `co.uk`
 // ===============================================================
 
 
-| <<ecs-group,client.user.group.*>>
```

Comment:
> In `generated/ecs/ecs_nested.yml`: chained reuses don't populate `nestings` all the way anymore. Example: `group` is reused in `user`, then `user` in `client`: client's `nestings` attribute will no longer list `client.user.group`.

Likewise with `trace.id`, this is related to this behavior change?

Reply:
Yes these are the chained nestings mentioned in the "caveats" of the PR.

I could work to add this back, but I'm not sure how useful this would be. So when we nest `group` in `user`, when looking at the nestings inside user, we correctly see that `group` is nested at `user.group`.

What we're losing (for now) in this PR is that when we turn around and nest `user` elsewhere like at `client.user`, we no longer list `group` as being nested under `client`. However we correctly list `user` as being nested under `client`.

My opinion is that someone seeing that `user` is nested there will realize that `user` comes with its own nesting of `group`.

Happy to work to re-introduce it, if you think it's important to list these chained nestings explicitly.

Reply:
Thanks for the explanation 😀

No I don't believe there's a current need to re-introduce; I agree that a user browsing the docs can still navigate logically and little is lost.